deployment/mmw_stack.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python
+"""Commands for setting up the Model My Watershed stack on AWS"""
+
+import argparse
+import os
+
+from cfn.stacks import build_stacks, get_config
+
+
+current_file_dir = os.path.dirname(os.path.realpath(__file__))
+
+
+def launch_stacks(mmw_config, aws_profile, **kwargs):
+    build_stacks(mmw_config, aws_profile, **kwargs)
+
+
+def main():
+    """Parse args and run desired commands"""
+    common_parser = argparse.ArgumentParser(add_help=False)
+    common_parser.add_argument('--aws-profile', default='default',
+                               help='AWS profile to use for launching stack '
+                                    'and other resources')
+    common_parser.add_argument('--mmw-config-path',
+                               default=os.path.join(current_file_dir,
+                                                    'default.yml'),
+                               help='Path to MMW stack config')
+    common_parser.add_argument('--mmw-profile', default='default',
+                               help='MMW stack profile to use for launching '
+                                    'stacks')
```

Comment:
Probably not that important, since once you learn the right way to use the CLI it isn't a problem anymore.